packages/perseus-editor/src/widgets/interactive-graph-editor.tsx

```diff
@@ -244,13 +246,18 @@ class InteractiveGraphEditor extends React.Component<Props> {
                     this.props.onChange({correct: correct});
                 },
             } as const;
+
+            // This is used to force a remount of the graph component
+            // when there's a significant change
+            const graphKey = `${correct.type}:${correct.numSegments || 0}`;
```

Comment:
This is specific to the `segment` graph, right? Will we need to adjust as we do other graph types?